app/src/split/bluetooth/peripheral.c

```diff
@@ -39,16 +41,30 @@ static const struct bt_data zmk_ble_ad[] = {
                   ),
     BT_DATA_BYTES(BT_DATA_UUID128_ALL, ZMK_SPLIT_BT_SERVICE_UUID)};
 
+static bool is_connected = false;
+
 static int start_advertising() {
     return bt_le_adv_start(BT_LE_ADV_CONN, zmk_ble_ad, ARRAY_SIZE(zmk_ble_ad), NULL, 0);
 };
 
+static void connected(struct bt_conn *conn, uint8_t err) {
+    is_connected = (err >= 0);
```

Comment:
The docs for this parameter say

> HCI error. Zero for success, non-zero otherwise.

Should positive values be treated as connected?

Reply:
Testing showed it working fine, but let me tweak to only use `0` as connected, and verify it works properly.

Reply:
Yup, still worked fine. Updated.